java/core/src/java/org/apache/orc/Reader.java

```diff
@@ -220,6 +243,7 @@ public Options(Configuration conf) {
      * Set the list of columns to read.
```

Comment:
Could you also add the tags to the Options class and its constructors?

Reply:
Sure. I added.